src/word.c

```diff
@@ -0,0 +1,67 @@
+#include "dictionary.h"
+#include <stdio.h>
+#include <strings.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>	
+
+int num_punctuation = 22;
+char* punctuation_array[] = {"+",",",":"," ",".","-","'","&","!","?",":",";","#","~","=","/","$","£","^","\n","_","<",">"};
+
+// checks if punctuation exists within array
+int is_in_array(char* punctuation_array[], char* word) {
+    int i = 0;
+	for ( ; i < num_punctuation ; i++) {
```

Comment:
Change this to `int i` and put `i = 0` where it should go in the for loop.

Reply:
done